sdk/include/platform/sunburst/platform-i2c.hh

```diff
@@ -233,6 +233,9 @@ struct OpenTitanI2c
 	  FifoControlTransmitReset = 1 << 8,
 	};
 
+	static constexpr uint32_t RXFIFODepth = 8;
+	static constexpr uint32_t FmtFIFODepth = 8;
```

Comment:
Is FmtFIFODepth used anywhere?

Reply:
I would be in favour of removing the `FmtFIFODepth` until it is used.

Reply:
Yeah I added it for completeness, not currently used so happy to remove it